blocks/api/registration.js

```diff
@@ -43,6 +43,13 @@ export function registerBlock( slug, settings ) {
 		);
 		return;
 	}
+	if ( true !== validateBlockSettings( settings ) ) {
```

Comment:
There was a similar discussion in the original validation introduced about how to name this function based on its behavior. Eventually it appears we inlined the validation. A few questions then:

- Should we inline block settings validation for consistency with the slug validation?
- Should we expect the `validate` to _do_ something, like `throw`, or perhaps prefixing with `is` to reflect its boolean return value is more accurate, like `isValidBlockSettings`
- If we know the function returns a boolean, do we need to be so explicit with the comparison, or can we simplify to `! isValidBlockSettings` ?